Packs/AWS/Integrations/AWS/AWS.py

```diff
@@ -749,6 +749,95 @@ def put_bucket_ownership_controls_command(client: BotoClient, args: Dict[str, An
+                }
+                readable_output = tableToMarkdown(
+                    name="Bucket Website Configuration",
+                    t=response.get("WebsiteConfiguration", {}),
+                    removeNull=True,
+                    headers=["ErrorDocument", "IndexDocument", "RedirectAllRequestsTo", "RoutingRules"],
+                    headerTransform=pascalToSpace,
+                )
+                return CommandResults(readable_output=readable_output,
+                                      outputs_prefix="AWS.S3-Buckets.BucketWebsite",
+                                      outputs=response.get("WebsiteConfiguration", {}),
+                                      raw_response=response.get("WebsiteConfiguration", {}))
+            raise DemistoException(
+                f"AWS S3 API call to get_bucket_website failed or returned an unexpected status code. "
+                f"Received HTTP Status Code: {response['ResponseMetadata']['HTTPStatusCode']}"
+            )
+        except Exception as e:
+            raise DemistoException(f"Error: {str(e)}")
+
+    @staticmethod
+    def get_bucket_acl_command(client: BotoClient, args: Dict[str, Any]) -> CommandResults:
+        """
+        Retrieves the Access Control List (ACL) of a specified Amazon S3 bucket.
+
+        The function calls the AWS S3 'get_bucket_acl' API to determine the permissions
+        granted to specific users or groups on the bucket.
+
+        Args:
+            client (BotoClient): The initialized Boto3 S3 client.
+            args: A dictionary containing arguments, expected to include 'bucket'
+                  (the name of the S3 bucket).
+
+        Returns:
+            CommandResults: An object containing the raw Access Control Policy details as
+                            outputs, and a md table summarizing the ACL configuration
+                            (Grants and Owner).
+        """
+        kwargs = {"Bucket": args.get("bucket")}
+        try:
+            response = client.get_bucket_acl(**kwargs)
+            if response["ResponseMetadata"]["HTTPStatusCode"] in [HTTPStatus.OK, HTTPStatus.NO_CONTENT]:
+                response["AccessControlPolicy"] = {
+                    "Grants": response.get("Grants"),
+                    "Owner": response.get("Owner"),
+                }
+                readable_output = tableToMarkdown(
+                    name="Bucket Acl",
+                    t=response.get("AccessControlPolicy", {}),
+                    removeNull=True,
+                    headers=["Grants", "Owner"],
+                    headerTransform=pascalToSpace,
+                )
+                return CommandResults(readable_output=readable_output,
+                                      outputs_prefix="AWS.S3-Buckets.BucketAcl",
+                                      outputs=response.get("AccessControlPolicy", {}),
+                                      raw_response=response.get("AccessControlPolicy", {}))
```

Comment:
Why is response["AccessControlPolicy"] required?
You can use response:
 ```suggestion
            if response["ResponseMetadata"]["HTTPStatusCode"] in [HTTPStatus.OK, HTTPStatus.NO_CONTENT]:
                readable_output = tableToMarkdown(
                    name="Bucket Acl",
                    t=response,
                    removeNull=True,
                    headers=["Grants", "Owner"],
                    headerTransform=pascalToSpace,
                )
                return CommandResults(readable_output=readable_output,
                                      outputs_prefix="AWS.S3-Buckets.BucketAcl",
                                      outputs=response,
                                      raw_response=response)
```
Please also add `outputs_key_field` to the CommandResults.